core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -117,18 +117,27 @@ public static void closeStreams(Closeable... streams) {
     // Added if to avoid NullPointerException in case one stream is being passed as null
     if (null != streams) {
       for (Closeable stream : streams) {
-        if (null != stream) {
-          try {
-            stream.close();
-          } catch (IOException e) {
-            LOGGER.error("Error while closing stream" + stream);
-          }
+        try {
+          closeStream(stream);
+        } catch (IOException e) {
+          LOGGER.error("Error while closing stream:" + e);
```

Comment:
But here you are not throwing any error. I don't see much difference from old and new code.